config/graph.arbitrum-one.yml

```diff
@@ -48,6 +48,8 @@ contracts:
     proxy: true
     init:
       controller: "${{Controller.address}}"
+    calls:
+      - fn: "syncAllContracts"
   EpochManager:
```

Comment:
🔍

Reply:
good catch, but I actually left the EpochManagers out on purpose, since they don't do any external calls... (I basically copied the list from test/lib/fixtures.ts)